mypy.ini

```diff
@@ -16,6 +16,22 @@ warn_unused_configs = True
 follow_imports = True
 ignore_errors = False
 
+[mypy-edb.pgsql.compiler.*]
+follow_imports = True
+ignore_errors = False
+# Equivalent of --strict on the command line:
+disallow_subclassing_any = True
+disallow_any_generics = True
+; disallow_untyped_calls = True
```

Comment:
Use # for comments and comment why you're disabling this. In checked.py the only reason is use of @overload which requires the default implementation to be untyped (this is where PEP 484 disagrees with Mypy I guess).

Reply:
VSCode did the `;` thing, not me :-)  `disallow_untyped_calls` is unset because the compiler calls out to lots of untyped code in `edb.pgsql`, which is beyond the scope here.